### PR TITLE
CIP-???? | Tune Parameters minPoolCost and K

### PR DESCRIPTION
**Pull request by ADARobinHood**
(see [rendered proposal in branch](https://github.com/ADARobinHood/CIPs/blob/master/CIP%20-%20%3F/README.md))

### CODE REVIEW

**Review by rphair — COMMENTED** [review]
@ADARobinHood it looks like this deprecates your own [CIP-0074](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0074) merged just 3 weeks ago.  If that's not what you want (i.e., if they should _both_ be concurrently Proposed or Active CIPs), would you please explain in this CIP, and summarise here in the discussion:
- how the earlier CIP relates to this one?
- particularly, how the further adjustment of `K` fits in with the Path to Active of the earlier CIP (which calls for a "Protocol Parameter Updates" but says nothing about changing `K`)?

File: CIP - ?/README.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+Making pledge more effective could also help solve the problem of exchanges, despite having no stake of their own, making >20% of the blocks. BNP is simply being greedy with their stake. It is more profitable to run their own pledge-less pools than to pay a fee to an SPO. Only a few public pools currently offer greater than baseline rewards, thanks to the imbalance mentioned above. If more pools could offer greater than baseline rewards, exchanges like Binance and Coinbase could choose to make more ADA by delegating to community pools.
```
Comment (TerminadaPool) [inline CIP - ?/README.md:78]: Large exchanges want to:
* Maximise rewards
* Maximise liquidity (minimise pledge and lock-up time)
* Maximise control (eg: preferential submission of their transactions, avoidance of over-saturating stake pools)

Increasing K will make pledge more effective.  Cardano's staking design forces exchanges to choose between increased pledge for increased rewards and reduced pledge for increased liquidity.  This proposal will shift the balance in the pledge direction somewhat and might push exchanges towards community staking.  However, staking to community pools will reduce their control over saturation, and increasing K would make this more difficult to manage.  Implementation of multi-pool delegation could help here?

**Comment by TerminadaPool** [comment]
I like this proposal better than CIP-0074 because I agree combining decreasing minPoolCost with increasing K is necessary for the reasons stated:
* Creates necessary "shake-up" to stimulate re-evaluation of pool choice
* Increases effectiveness (value) of pledge

**Comment by ADARobinHood** [comment]
> I like this proposal better than CIP-0074 because I agree combining decreasing minPoolCost with increasing K is necessary for the reasons stated:
> 
> * Creates necessary "shake-up" to stimulate re-evaluation of pool choice
> * Increases effectiveness (value) of pledge

I agree that both a reduction (elimination) of minPoolCost and raising K is what's needed. My planned course was to propose CIP-0074 - Eliminate minPoolCost, then propose a separate CIP for a K increase. CIP-0074 HAS to be a prerequisite to the K increase. 

I figured that by proposing them separately they would be more likely to get adopted as opposed to proposing 2 large parameter adjustments at the same time.

**Review by rphair — COMMENTED** [review]
> @ADARobinHood: This CIP is not mean to deprecate CIP-0074. ... to give options and provide rational for different thoughts on different changes ...

OK, then like CIP-0074 **this CIP will also need a _Path to Active_ section** in order to merge it as `Proposed` if they are both to exist in parallel.  If you think (or if it's decided in this discussion) that this CIP depends on CIP-0074, that's the place to mention it... likewise you can update the _Path to Active_ of CIP-0074 if it involves either consideration or implementation of this one. 😎

**Review by rphair — DISMISSED** [review]
@ADARobinHood this PR is coming up for "Triage" (initial presentation & cataloguing) at the Discord CIP meeting in about 90 minutes.  Upon further consideration I'm going to recommend as I briefly suggested earlier in https://github.com/cardano-foundation/CIPs/pull/387#pullrequestreview-1199767540 that this proposal and CIP-0074 be merged somehow: if not deprecating CIP-0074 outright since it is only a rudimentary version of this one.

> alternative CIPs are meant to give options and provide rational for different thoughts on different changes. Community awareness, discussion, and mostly education is important on these concepts. Both CIPs give a window into the future if that specific CIP is chosen to implement.

Actually, that is what:
- ... a _single_ CIP is for: if two proposals have the same goal, with a nearly identical solution and other essential components (even the same author in this case).  If so it may have a complex _Path to Active_ that depends upon input from company stakeholders, or the observation of gradually varying some initial parameters: including some "forks in the road" like you have between this & CIP-0074.
- ... a single _CPS_ is for: if the solution is declared to be undecided (as you involuntarily argue for in this case, by presenting two divergent CIPs on the same subject).  I'd say this is the least attractive option if you're concerned about adoption, because the less concrete presentation tends to delay adoption a bit & many in the community consider this to be urgent.

In either case, a single document is what would best focus attention on the problem at hand: _not_ wallpapering the CIP space with functionally almost-equivalent proposals in hope of widening the path so that the protocol developers follow.

I'll ask editors @KtorZ @SebastienGllmt @crptmppt briefly how they feel about having these as 2 separate CIPs and will post here if there is any consensus otherwise & if necessary will update my review status accordingly.  Only "Triage" is scheduled this meeting: but by the time a detailed review comes up I would suggest in open discussion that this PR be rewritten _as an update_ to CIP-0074 if another option to consolidate these 2 proposals is not taken.

**Comment by rphair** [comment]
In light of today's CIP meeting discussion in the last hour, I'm demoting my last "blocking" review to an ordinary comment.  Though I still believe your submission would work better as a single, combined CIP than two proposals, we (mainly including @KtorZ) agreed that different presentations of parameter sets are _not_ the first thing the community needs to address this problem: but rather a functional dialogue and process between IOG and the stakeholders, operators and developers who might rely on these parameter changes.

Effectively all these RSS (Reward Sharing Scheme) proposals are in limbo until IOG produces, or accepts, something like a [Cardano Problem Statement](https://github.com/cardano-foundation/CIPs/tree/master/CIP-9999) (CPS) which provides a framework to review RSS proposals.  This has to be the first step in considering anything like your proposal.

I haven't seen you in this so far, but there's a Discord in which the efforts to engage IOG have been discussed along with other particular RSS schemes.  Since we can't do anything to progress his PR until we have that framework, it might help to post any ideas about that here (please post or message me if you need an invite to this server... this thread also contains a link to similar discussions in an MBO-like group on Matrix): https://discord.com/channels/971785110770831360/971785110770831366

**Comment by gitmachtl** [comment]
I would like to see things like a minPoolFee implemented together with such changes.

File: CIP - ?/README.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
+---
+CIP: ?
```
Comment (KtorZ) [inline CIP - ?/README.md:2]:
```suggestion
CIP: 83
```

**Comment by rphair** [comment]
Closing after 3½ months of no activity and no word from author since end of 2022.  Author pressed https://github.com/cardano-foundation/CIPs/pull/358 through to merging but left this one behind though I think this proposal is more comprehensive.

Now that RSS proposals have been granted Ledger status as updated across the board in https://github.com/cardano-foundation/CIPs/pull/536 this one would be ready to merge also if the author is still interested, though we might assume abandonment because there was no response to tagging @ADARobinHood in that PR.

Consequently if any constructive interest from the author, or another author/advocate that will take over editing and review, we can reopen this.

File: CIP - ?/README.md (hunk 2 above)
Comment (rphair) [reply CIP - ?/README.md:2]: Removing number from PR title due to having been officially assigned to a proposal which was more active in this period: https://github.com/cardano-foundation/CIPs/blob/master/CIP-0083/README.md

**Comment by rphair** [comment]
Retitling to remove `83` as per https://github.com/cardano-foundation/CIPs/pull/387#discussion_r1266567536 - if this gets going again a new number will have to be assigned.

**Pull request closed without merging**